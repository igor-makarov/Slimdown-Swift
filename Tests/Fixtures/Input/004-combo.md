# Page title

And **now** for something _completely_ different.

Also include ~~deleted text~~ and :"inline quotation":.