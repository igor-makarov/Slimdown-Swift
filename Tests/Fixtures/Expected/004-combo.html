<h1>Page title</h1>

<p>And <strong>now</strong> for something <em>completely</em> different.</p>

<p>Also include <del>deleted text</del> and <q>inline quotation</q>.</p>